src/System.Net.Sockets/src/System/Net/Sockets/BaseOverlappedAsyncResult.Windows.cs

```diff
@@ -107,17 +107,16 @@ private static unsafe void CompletionPortCallback(uint errorCode, uint numBytes,
                             bool success = Interop.Winsock.WSAGetOverlappedResult(
                                 socket.SafeHandle,
                                 asyncResult._nativeOverlapped,
-                                out numBytes,
+                                &numBytes,
                                 false,
-                                out ignore);
-                            if (!success)
-                            {
-                                socketError = SocketPal.GetLastSocketError();
-                            }
+                                &ignore);
+
                             if (success)
                             {
-                                NetEventSource.Fail(asyncResult, $"Unexpectedly succeeded. errorCode:{errorCode} numBytes:{numBytes}");
+                                NetEventSource.Fail(asyncResult, $"WSAGetOverlappedResult unexpectedly succeeded. errorCode:{errorCode} numBytes:{numBytes}");
```

Comment:
Nit: As long as you're using the function name, use `nameof(Interop.Winsock.WSAGetOverlappedResult)`?